.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```

Comment:
We should create another PR to enable eslint / pretters tabWidth option and clean up the project